Tests/Admin/AdminHelperTest.php

```diff
@@ -18,6 +18,9 @@
 
 class AdminHelperTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * @covers Sonata\AdminBundle\Admin\AdminHelper::getChildFormBuilder
+     */
```

Comment:
`testA` is not calling `a` or `c`

With **and** without `@covers`annotation, no coverage is done.

Let me explain it a little bit more in detail!
```php
<?php

class Foo {
    public function a ()
    {
       $this->b();
       $this->c();
    }
    public function b ()
    {
    }
    private function c ()
    {
    }
}
```

```php
class FooTest {
    /**
     *
     * @covers a
     * @covers c
     */
    public function testA()
    {
        $this->assert('test', $this->a());
    }
```

We got the following coverage:
* coverage for Foo::a()
* NO coverage for Foo::a(), because i didn't annotate it
* coverage for Foo::c()

But i would not set `@covers c`, because maybe i didn't test ALL the edge cases of `c()`, so this would be better:

```php
class FooTest {
    /**
     *
     * @covers a
     */
    public function testA()
    {
        $this->assert('test', $this->a());
    }
```

Now i am sure that i need to write a test for `b()` and `c()`, only callings some code lines is not a real test coverage for me and with the `@covers` annotation we can aschieve this more clear!